.travis.yml

```diff
@@ -56,6 +56,12 @@ matrix:
       sudo: required
     - python: "2.7"
       env: TOXENV=nginxroundtrip
+    - language: generic
```

Comment:
I created #5381 to document our tests in the `test-everything` branch, but I don't think want to readd these tests here. Since Travis macOS tests take so long to run, we've been doing this and other testing in the `test-everything` branch which is run nightly.

We should just merge the fix above and once that has landed, merge the fix into `test-everything`.